.npmignore

```diff
@@ -0,0 +1,10 @@
+/bin
+/node_modules
+/test
+.editorconfig
+.gitignore
+.jshintignore
+.jshintrc
+.npmignore
+.travis.yml
+CONTRIBUTING.md
```

Comment:
Do not remove `CONTRIBUTING.md`. `node_modules` is [handled by npm automatically](https://docs.npmjs.com/misc/developers). Infact, it would be better to keep it minimal. `test` is the main thing we want to avoid packaging in.